### PR TITLE
CLI: check state dump time for mainnet

### PR DESCRIPTION
**Pull request by AliceInHunterland**
It is an example of traverse MPT implementation. We do not need this command.

### CODE REVIEW

File: cli/server/mptdump.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+// TraverseMPT collects key-value pairs from the TrieStore and returns them.
+func TraverseMPT(store *mpt.TrieStore) ([]KVPair, error) {
+	var kvPairs []KVPair
+	prefix := []byte{byte(storage.STStorage)}
+	rng := storage.SeekRange{Prefix: prefix}
+
+	store.Seek(rng, func(k, v []byte) bool {
+		kvPairs = append(kvPairs, KVPair{
```
Comment (roman-khimov) [inline cli/server/mptdump.go:30]: Please don't try to store it all in memory. It can be huge. Write to some output immediately.

**Comment by roman-khimov** [comment]
What's the time with the current code?

**Comment by AliceInHunterland** [comment]
> What's the time with the current code?

it takes 6 minutes on my laptop if everything is correct.

**Comment by roman-khimov** [comment]
OK, this looks legit and we're not currently interested in this particular command. It's not super-efficient (binary dump would be a bit more appropriate), but if takes some minutes then I'm completely satisfied with it. Please add time/size into #3519.

**Pull request closed without merging**